app/assets/stylesheets/editor.css

```diff
@@ -32,17 +32,17 @@ pre {
     background-color: #f5f5f5;
     border: 1px solid #ccc;
     border-radius: 4px;
+  }
 
-  // Account for some code outputs that place code tags in pre tags
+  /* Account for some code outputs that place code tags in pre tags */
   code {
-    padding: 0;
+    padding: 0px;
     font-size: inherit;
     color: inherit;
     white-space: pre-wrap;
     background-color: transparent;
     border-radius: 0;
   }
-}
 
```

Comment:
I believe this was an extra close curly-brace that may have been affecting the CSS! Also fixed the commenting before this block.